### PR TITLE
updated to puppeteer 5.2

### PR DESCRIPTION
**Pull request by jnd0**
Updated to the last Puppeteer available version

### CODE REVIEW

**Comment by vladgolubev** [comment]
Thanks!

Could you please commit `.zip` file using Git LFS? 

See https://git-lfs.github.com/

B/c GitHub now thinks file size increased by 350000%

image

**Comment by jnd0** [comment]
I think I've done on the last commit

File: .gitattributes [hunk 1]
```diff
@@ -1,3 +1,4 @@
 * text=auto
 *.js text eol=lf
 *.zip filter=lfs diff=lfs merge=lfs -text
+chrome_aws_lambda.zip filter=lfs diff=lfs merge=lfs -text
```
Comment (vladgolubev) [inline .gitattributes:4]: Thanks for adding this line!

**Comment by vladgolubev** [comment]
New layer version published:

```
arn:aws:lambda:us-east-1:764866452798:layer:chrome-aws-lambda:19
```

see readme for the rest of the regions

**Comment by jnd0** [comment]
Thank you so much for publishing this. 

I've tried it and unfortunately doesn't work. 

I get the following output when trying to execute this latest version of the layer 
```
{
  "code": "MODULE_NOT_FOUND",
  "requireStack": [
    "/opt/nodejs/node_modules/chrome-aws-lambda/source/puppeteer/lib/Browser.js",
    "/opt/nodejs/node_modules/chrome-aws-lambda/source/index.js",
    "/var/task/index.js",
    "/var/runtime/UserFunction.js",
    "/var/runtime/index.js"
  ]
```
 I think is related to this: https://github.com/alixaxel/chrome-aws-lambda/pull/157 

Should've checked before creating the merge request. Sorry 

I will open a new one as soon as this gets fixed.

**Comment by vladgolubev** [comment]
Np @jnd0! I'll publish your fix once the upstream issue is resolved

**Comment by jnd0** [comment]
Created merge https://github.com/shelfio/chrome-aws-lambda-layer/pull/24

This will fix the mess that has been created here as well as updating Puppeteer to the latest available version,

It will work this time I've checked manually creating my own layer before creating this pull request.

Thanks again @vladgolubev this was not too bad in the end for my first ever public merge request on GitHub 😅

**Pull request merged**